### ISSUE TITLE
Alternative Download source?

### ISSUE THREAD

**Issue by IzzySoft** [skilion/sleep-timer#1]
Currently your app is only available via Google's walled garden (aka Playstore). Would you consider making it available to the "free folks" outside its walls (i.e. those who for some reason either cannot or wantnot use it)? I think e.g. of attaching it to [tags](https://github.com/skilion/sleep-timer/releases) here in your repo, or to have [F-Droid](https://f-droid.org/) package and include it with their catalog.

What do you think, @skilion?

**Comment by swltr** [comment]
@skilion Your sleep timer app is simple but really useful (at least to me.) It would be really cool if it could be added to the F-Droid app store for those among us who prefer to use Google-free devices.

Would you approve its inclusion there? As far as I can see, it meets all their [inclusion criteria](https://f-droid.org/docs/Inclusion_Policy/). (Having a Git tag for the current release would be helpful, though. Could you add that?)

This doesn't mean that it would be you who would have to do [the actual work](https://f-droid.org/docs/Submitting_to_F-Droid_Quick_Start_Guide/). Packaging for F-Droid could also be done by someone else, but it would be helpful to see that you're not opposed to it.

**Comment by IzzySoft** [comment]
> This doesn't mean that it would be you who would have to do [the actual work](https://f-droid.org/docs/Submitting_to_F-Droid_Quick_Start_Guide/). Packaging for F-Droid could also be done by someone else, but it would be helpful to see that you're not opposed to it.

Right. The linked guide is with icing on the cake. The "lazy/easy" way you can [find outlined here](https://android.izzysoft.de/articles/named/fdroid-intro-2#rfp). I hope you're still maintaining the app, @skilion – as this issue wasn't replied to since I've opened it almost 3 years ago?
